includes/wc-admin-update-functions.php

```diff
@@ -41,3 +41,20 @@ function wc_admin_update_0201_order_status_index() {
 function wc_admin_update_0201_db_version() {
 	Installer::update_db_version( '0.20.1' );
 }
+
+/**
+ * Rename "gross_total" to "total_sales".
+ * See: https://github.com/woocommerce/woocommerce-admin/issues/3175
+ */
+function wc_admin_update_0210_rename_gross_sales() {
+	global $wpdb;
+
+	$wpdb->query( "ALTER TABLE {$wpdb->prefix}wc_order_stats CHANGE gross_total total_sales DOUBLE" );
```

Comment:
This just added `total_sales` with all zeroes instead of renaming `gross_total` which still exists. Is this on purpose?

Reply:
No, the intended result is that the column is renamed and the data is the same. What MySQL version are you using?

Reply:
Fixed in 551cc31 - though, it's kind of hacky since we shouldn't _really_ be renaming columns in WP (breaks back compat). I could argue that's OK with WC-Admin since we're a feature plugin and haven't made it to core WC yet.

Reply:
> I could argue that's OK with WC-Admin since we're a feature plugin and haven't made it to core WC yet.

We probably don't see very many people going back to an earlier version.